ovos_plugin_common_play/ocp/res/ui/OVOSAudioPlayer.qml

```diff
@@ -341,6 +343,17 @@ Mycroft.Delegate {
                 anchors.leftMargin: Kirigami.Units.largeSpacing
                 anchors.rightMargin: Kirigami.Units.largeSpacing
 
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        if(root.spectrumType == 1) {
```

Comment:
can you make this emit a bus event reporting type?

`Message("ovos.common_play.spectrum", {"type": 1})`

Reply:
I have added a global bus message, as not sure which one you were referring to, if required i can change it to a gui event